src/main/java/com/epam/test_generator/controllers/JiraController.java

```diff
@@ -0,0 +1,91 @@
+package com.epam.test_generator.controllers;
+
+import com.epam.test_generator.pojo.JiraProject;
+import com.epam.test_generator.pojo.JiraStory;
+import com.epam.test_generator.services.JiraService;
+import io.swagger.annotations.ApiImplicitParam;
+import io.swagger.annotations.ApiImplicitParams;
+import net.rcarz.jiraclient.JiraException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+/**
+ * Allow communication with Jira
+ */
+@RestController
+public class JiraController {
+
+    @Autowired
+    private JiraService jiraService;
+
+    @Secured({"ROLE_ADMIN", "ROLE_TEST_LEAD"})
+    @ApiImplicitParam(name = "Authorization", value = "add here your token", paramType = "header", dataType = "string", required = true)
+    @RequestMapping(value = "/createSuits", method = RequestMethod.PUT, consumes = "application/json")
+    public ResponseEntity<String> createStoriesForProject(@RequestBody List<JiraStory> jiraStories) throws JiraException {
+
+        jiraService.addStoriesToExistedProject(jiraStories);
+        return new ResponseEntity<>(HttpStatus.OK);
+    }
+
+    @Secured({"ROLE_ADMIN", "ROLE_TEST_LEAD"})
+    @ApiImplicitParam(name = "Authorization", value = "add here your token", paramType = "header", dataType = "string", required = true)
+    @RequestMapping(value = "/createProject", method = RequestMethod.PUT, consumes = "application/json")
```

Comment:
I'd like to see this controller corresponds to CRUD design pattern. Let's discuss it off line tomorrow.